### PR TITLE
[WIP] Merge 1 4 into master

### PR DESCRIPTION
**Pull request by jjanvier**

### CODE REVIEW

File: src/Pim/Bundle/DataGridBundle/Datagrid/Configuration/Product/ContextConfigurator.php [hunk 1]
```diff
@@ -284,6 +296,41 @@ protected function getCurrentLocaleCode()
...
+    }
+
+    /**
+     * @return array
+     */
+    protected function getAttributeIdsFromProductGroupAxes()
+    {
+        $attributeIds = [];
+
+        if ((null !== $productGroupId = $this->getProductGroupId()) && null !== $this->productGroupRepository) {
```
Comment (skeleton) [inline src/Pim/Bundle/DataGridBundle/Datagrid/Configuration/Product/ContextConfigurator.php]: remove condition `null !== $this->productGroupRepositor`

File: src/Pim/Bundle/DataGridBundle/Datagrid/Configuration/Product/ContextConfigurator.php [hunk 2]
```diff
@@ -284,6 +297,41 @@ protected function getCurrentLocaleCode()
...
+    {
+        $productGroupId = null;
+        if (null !== $productGroup = $this->request->get('group', null)) {
+            $productGroupId = $productGroup->getId();
+        }
+        if (null === $productGroupId) {
+            $productGroupId = $this->requestParams->get('currentGroup', null);
+        }
+
+        return $productGroupId;
```
Comment (ClementGautier) [inline src/Pim/Bundle/DataGridBundle/Datagrid/Configuration/Product/ContextConfigurator.php]: I prefer
```php
return $this->request->get('group', $this->requestParams->get('currentGroup'))
```

Comment (ClementGautier) [inline src/Pim/Bundle/DataGridBundle/Datagrid/Configuration/Product/ContextConfigurator.php]: and, as it is a one line function I would prefer to remove it as it is used only in getAttributeIdsFromProductGroupAxes

Comment (ClementGautier) [inline src/Pim/Bundle/DataGridBundle/Datagrid/Configuration/Product/ContextConfigurator.php]: (BTW I know that its not your code, fix it if you agree / have time)

**Comment by ClementGautier** [comment]
GTM

File: src/Pim/Bundle/DataGridBundle/Datagrid/Configuration/Product/ContextConfigurator.php (hunk 1 above)
Comment (jjanvier) [inline src/Pim/Bundle/DataGridBundle/Datagrid/Configuration/Product/ContextConfigurator.php]: done

File: src/Pim/Bundle/DataGridBundle/Datagrid/Configuration/Product/ContextConfigurator.php (hunk 2 above)
Comment (jjanvier) [inline src/Pim/Bundle/DataGridBundle/Datagrid/Configuration/Product/ContextConfigurator.php]: that was my code, fixed

**Comment by jmleroux** [comment]
no more `/` in branch name please.

**Pull request closed without merging**